src/vivarium_gates_nutrition_optimization/data/loader.py

```diff
@@ -381,6 +382,35 @@ def get_moderate_hemorrhage_probability(key: str, location: str) -> pd.DataFrame
 
     return moderate_hemorrhage_probability
 
+###########################
+# Background Morbidity    #
+###########################
+
+def load_background_morbidity(key: str, location: str) -> pd.DataFrame:
+    all_cause_yld_rate = extra_gbd.get_all_cause_yld_rate(location)
+    all_cause_yld_rate = all_cause_yld_rate[vi_globals.DEMOGRAPHIC_COLUMNS + vi_globals.DRAW_COLUMNS]
+    all_cause_yld_rate = reshape_to_vivarium_format(all_cause_yld_rate, location)
+
+    all_anemia_yld_rate = extra_gbd.get_anemia_yld_rate(location)
+    all_anemia_yld_rate = all_anemia_yld_rate.groupby(vi_globals.DEMOGRAPHIC_COLUMNS)[vi_globals.DRAW_COLUMNS].sum().reset_index()
```

Comment:
Why are you summing demographic groups? You don't seem to be doing this anywhere else

Reply:
GBD returns these data for each included sequela separately, so to combine them we need to group and sum them. In `extra_gbd`, the GBD call for, for example, the anemia sequelae of maternal hemorrhage also does this operation.

Reply:
So I think something different is happening here for the anemia impairment - sorry I didn't think to specify this in the documentation. The GBD call returns anemia impairment YLDs due to all different causes with GBD. We want anemia due to all causes, so we should restrict to `anemia_yld_rate = anemia_yld_rate.loc[anemia_yld_rate.cause_id==294]` for line 395 instead.

Reply:
I think there is a way that you can structure the call from GBD so that it only returns impairment YLDs due to a specified cause ID, but the above will work just as well!